airbyte-cdk/python/unit_tests/test/mock_http/test_request.py

```diff
@@ -18,40 +18,100 @@ def test_given_query_params_in_url_and_also_provided_then_raise_error(self):
 
     def test_given_same_url_query_params_and_subset_headers_when_matches_then_return_true(self):
         request_to_match = HttpRequest("mock://test.com/path", {"a_query_param": "q1"}, {"first_header": "h1"})
-        request_received = HttpRequest("mock://test.com/path", {"a_query_param": "q1"}, {"first_header": "h1", "second_header": "h2"})
-        assert request_received.matches(request_to_match)
+        actual_request = HttpRequest("mock://test.com/path", {"a_query_param": "q1"}, {"first_header": "h1", "second_header": "h2"})
+        assert actual_request.matches(request_to_match)
 
     def test_given_url_differs_when_matches_then_return_false(self):
         assert not HttpRequest("mock://test.com/another_path").matches(HttpRequest("mock://test.com/path"))
 
     def test_given_query_params_differs_when_matches_then_return_false(self):
         request_to_match = HttpRequest("mock://test.com/path", {"a_query_param": "q1"})
-        request_received = HttpRequest("mock://test.com/path", {"another_query_param": "q2"})
-        assert not request_received.matches(request_to_match)
+        actual_request = HttpRequest("mock://test.com/path", {"another_query_param": "q2"})
+        assert not actual_request.matches(request_to_match)
 
     def test_given_query_params_is_subset_differs_when_matches_then_return_false(self):
         request_to_match = HttpRequest("mock://test.com/path", {"a_query_param": "q1"})
-        request_received = HttpRequest("mock://test.com/path", {"a_query_param": "q1", "another_query_param": "q2"})
-        assert not request_received.matches(request_to_match)
+        actual_request = HttpRequest("mock://test.com/path", {"a_query_param": "q1", "another_query_param": "q2"})
+        assert not actual_request.matches(request_to_match)
 
     def test_given_headers_is_subset_differs_when_matches_then_return_true(self):
         request_to_match = HttpRequest("mock://test.com/path", headers={"first_header": "h1"})
-        request_received = HttpRequest("mock://test.com/path", headers={"first_header": "h1", "second_header": "h2"})
-        assert request_received.matches(request_to_match)
+        actual_request = HttpRequest("mock://test.com/path", headers={"first_header": "h1", "second_header": "h2"})
+        assert actual_request.matches(request_to_match)
 
     def test_given_headers_value_does_not_match_differs_when_matches_then_return_false(self):
         request_to_match = HttpRequest("mock://test.com/path", headers={"first_header": "h1"})
-        request_received = HttpRequest("mock://test.com/path", headers={"first_header": "value does not match"})
-        assert not request_received.matches(request_to_match)
+        actual_request = HttpRequest("mock://test.com/path", headers={"first_header": "value does not match"})
+        assert not actual_request.matches(request_to_match)
+
+    def test_given_same_body_mappings_value_when_matches_then_return_true(self):
```

Comment:
I moved the tests you had done in `test_http_mocker` to this class to avoid having duplicates and making sure it's as unit as a unit test can be